src/utils/cli.utils.ts

```diff
@@ -1,6 +1,10 @@
 import { ux } from '@oclif/core';
 
 export class CLIUtils {
+  static log(message: string) {
+    ux.log(ux.colorize('green', message));
```

Comment:
you have `this.log` in the command instance, why green tho? is the same as `CLIUtils.success`